source/chapters/user_interface.rst

```diff
@@ -605,29 +606,55 @@ Headphone and Master Mix Controls
             want to listen to in your headphones.
 
-**Head volume Knob**
-  This button adjusts your headphone volume. You can adjust the volume of a
-  single deck's signal you are listening to in the headphones with the
-  :ref:`Gain <interface-eq-gain>` knob.
-
 **Head-Split Button**
-  If activated, two mono signals are send to the headphone output instead of one
+  If activated, two mono signals are sent to the headphone output instead of one
   stereo signal. The master signal plays in the right channel, while the cueing
-  signal plays in the left channel. You can still adjust the head-mix so in the
-  left channel is not the pure cueing signal.
+  signal plays in the left channel. You can still adjust the
+  :guilabel:`Head-Mix Knob` to control the mix of the master and cue signals in
+  the left channel.
 
 **Balance Knob**
   This knob allows you to adjust the :term:`balance` (left/right orientation) of
   the master output.
 
-**Volume Knob**
-  The Volume Knob controls the overall volume of the master output. Adjust
-  this knob so that the :ref:`Master VU meters <interface-vu-meter>` are just
-  at the peak.
-
-  .. hint:: If the Peak indicator on top of the Master VU meter flashes, the
-            master output signal is clipping (too loud). Lower the volume with
-            using the volume knob.
+.. _interface-gain-knob:
+  
+**Gain Knob**
+  The Gain Knob controls the gain of the signal that Mixxx sends to your
+  sound card. Generally, this knob should not be adjusted. To change the output
+  volume, a gain control should be adjusted as close to the speakers as possible
+  for the best signal-to-noise ratio throughout your signal chain. Ideally, the
+  volume would only be adjusted by adjusting the gain on the power amplifier.
+  On active speakers, including computer speakers, studio monitors, and PA
+  systems that do not require a separate amplifier, this means adjusting the
+  gain on the speakers (which may be labeled "Volume").
+  
+  As a DJ, you may not have access to the power amplifier. In that case,
+  adjust the volume by adjusting the gain on the equipment closest to the power
+  amplifier that you have access to. Typically, this is a DJ mixer. If you are
+  not going through a mixer, adjust the gain of your sound card. If your sound
+  card does not have any controls on it (for example, for sound cards built into
+  computers), adjust the sound card settings with your :term:`OS <operating system>`
+  mixer program. The Master Gain Knob in Mixxx should be a last resort for
+  adjusting volume.
+  
+  .. warning:: **In no case should any part of the signal chain be clipping,
+               indicated by a level meter going into its red region
+               or an LED labeled "clipping" turning on.**
+  
+  .. seealso:: For a more in depth explanation of how to set your gains, see
+               :ref:`Setting your levels properly (gain staging) <djing-gain-staging>`.
```

Comment:
I intentionally avoided writing about specific numeric values for levels because of that bug.